draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -1119,21 +1114,115 @@ SDP Answer
           from SSRC to identification-tag is received. Note that the mapping
           from SSRC to identification-tag can change at any time during an RTP
           session.
+	</t>
+
+
+      <t>Note: The following algorithm does not yet have WG consensus
+      but is included here as something concrete for the working group
+      to discuss.</t>
+
+      <t>To prepare for demultiplexing RTP packets to the correct "m="
+      line, the following steps MUST be followed for each BUNDLE
+      group.</t>
+
+      <t>
+        <list>
+
+          <t>Construct a table mapping MID to "m=" line for each
+          "m=" line in this BUNDLE group.</t>
+
+          <t>Construct a table mapping incoming SSRC to "m=" line for
+          each "m=" line in this BUNDLE group and for each SSRC
+          configured for receiving in that "m=" line.</t>
+
+          <t>Construct a table mapping outgoing SSRC to "m=line" for
+          each "m=" line in this BUNDLE group and for each SSRC
+          configured for sending in that "m=" line.</t>
+
+          <t>Construct a table mapping payload type to "m=" line for
+          each "m=" line in the BUNDLE group and for each payload type
+          configured for receiving in that "m=" line.  If any payload
+          type is configured for receiving in more than one "m=" line
+          in the BUNDLE group, do not it include it in the table.</t>
+        </list>
+      </t>
+
+      <t>As "m=" lines are added or removed from the BUNDLE groups, or
+      their configurations are changed, the tables above must also
+      be updated.</t>
+
+      <t>For each RTP packet received, the following steps MUST be
+      followed to route the packet to the correct "m=" section within
+      a BUNDLE group:</t>
+
+      <t>
+        <list>
+          <t>If the packet has a MID and that MID is not in the table
+          mapping MID to "m=" line, drop the packet and stop.</t>
+
+          <t>If the packet has a MID and that MID is in the table
+          mapping MID to "m=" line, update the incoming SSRC mapping
+          table to include an entry that maps the packet's SSRC to the
+          "m=" line for that MID.</t>
+
+          <t>If the packet's SSRC is in the incoming SSRC mapping
+          table, route the packet to the associated "m=" line and
+          stop.</t>
+
+          <t>If the packet's payload type is in the payload type
+          table, update the the incoming SSRC mapping table to include
+          an entry that maps the packet's SSRC to the
+          "m=" line for that payload type. In addition, route the packet
+          to the associated "m=" line and stop.</t>
+
+          <t>Otherwise, drop the packet.</t>
+        </list>
+      </t>
+
+      <t>For each RTCP packet received (including each RTCP packet that is part
+      of a compound RTCP packet), the following type-specific handling
+      MUST be performed to route the packet:</t>

```

Comment:
As I said in Seoul, I don't think claiming to have an exhaustive list of the RTCP handlers needed in this spec is a Good Thing. Suggested reformulation:

", the packet MUST be routed to the appropriate handler for the SSRCs it contains information about. Some examples of such handling are given below."

This makes it clear to implementors that when they encounter (for example) TMMBR packets, they have to route them reasonably, even though TMMBR is not listed below.